.env.test

```diff
@@ -93,11 +93,7 @@ SIMPLE_SERVER_HOST_PROTOCOL='https'
 #
 # Country specific configuration
 #
-ANALYTICS_TIME_ZONE='Asia/Kolkata'
-CRON_TIME_ZONE='Asia/Kolkata'
 DEFAULT_COUNTRY=IN
```

Comment:
Shall we make this full country name to avoid ambiguity?

Reply:
I'm ok leaving this as a country code. The code is still being used in some places too (specifically phone number parsing, I think).